src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+}
+
+export function getLatestTodoId(arr: Todo[]): number {
+  const ids: number[] = arr.map(todo => todo.id);
+
+  return Math.max(...ids);
+}
```

Comment:
How does this relate to the types that are located in this file. Please move this function to another file, utils for example